posthog/settings/temporal.py

```diff
@@ -27,6 +27,7 @@
 )
 
 CLICKHOUSE_MAX_EXECUTION_TIME: int = get_from_env("CLICKHOUSE_MAX_EXECUTION_TIME", 0, type_cast=int)
+CLICKHOUSE_MAX_MEMORY_USAGE: int = get_from_env("CLICKHOUSE_MAX_MEMORY_USAGE", 0, type_cast=int)
```

Comment:
Let's have a 100GB limit just in case
```suggestion
CLICKHOUSE_MAX_MEMORY_USAGE: int = get_from_env("CLICKHOUSE_MAX_MEMORY_USAGE", 100*1000*1000*1000, type_cast=int)
```